user_interface_components/dialog_wrapper.md

```diff
@@ -55,7 +55,7 @@ In this case, you bind a UI Designer form to your class extending
 [DialogWrapper](upsource:///platform/platform-api/src/com/intellij/openapi/ui/DialogWrapper.java),
 bind the top-level panel of the form to a field and return that field from the `createCenterPanel()` method.
 
-To display the dialog, you call the `show()` method and then use the `getExitCode()` method to check how the dialog was closed.
+To display the dialog, call the `showAndGet()` method, which waits until the dialog is closed and returns its exit code.
```

Comment:
Could you please leave original sentence and add your suggestion as additional "common" case? Thanks.

Reply:
I've updated my changes. Is this what you mean?